receiver/lokireceiver/internal/unmarshal_test.go

```diff
@@ -0,0 +1,128 @@
+package internal
```

Comment:
please add the copyright comment at the top of the file.

Reply:
done